insonmnia/locator/storage/inmemory/storage.go

```diff
@@ -0,0 +1,97 @@
+package inmemory
```

Comment:
I don't see where do we use it

Reply:
You are right. It was used when there was only inmemory storage. We should decided where to support both inmemory/libkv or delete the former and leave the later only.